.vscode/launch.json

```diff
@@ -37,7 +37,9 @@
                 "--annotations_file", "filtered.csv",
                 "--crop_box", "130", "275", "140", "140",
                 "--log_transformed_images", "False",
-                "--dropout_p", "0.3",
+                "--n_conf_layers", "3",
```

Comment:
3 layers of configuration?  😆 

There was a way to suggest a change but I can't find it

Reply:
I keep getting that wrong!